samples/request-response-mediation-interceptors/k8s-resources/api.yaml

```diff
@@ -24,6 +24,11 @@ spec:
   apiType: REST
   apiVersion: 1.0.0
   context: /interceptor/1.0.0
+  apiProperties:
```

Comment:
Shall we change the name to `property` as these properties are attached in API level? Use singular nouns for the property names. Refer https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/